indico/web/client/js/custom_elements/ind_menu.js

```diff
@@ -0,0 +1,57 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2023 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import './ind_menu.scss';
+import {domReady} from 'indico/utils/domstate';
+
+let lastId = 0; // Track the assigned IDs to give each element a unique ID
+
+customElements.define(
+  'ind-menu',
+  class extends HTMLElement {
+    connectedCallback() {
+      domReady.then(() => {
+        const trigger = this.querySelector('button');
+        const list = this.querySelector('menu');
+
+        console.assert(
+          trigger.nextElementSibling === list,
+          'The <menu> element must come after <button>'
+        );
+
+        trigger.setAttribute('aria-expanded', false);
```

Comment:
I guess this should be `'false'` since AFAIK DOM attributes are always strings? (even though the `false` is probably implicitly converted to a string

Reply:
Yes, it's always converted to string, and that is negligible performance hit too, so you'd just waste two characters to be pedantic. :)

Reply:
Just asking because you (correctly) check for `!== 'true'` below and I could already see a less experienced developer checking for `=== false` for whatever reason (after all, it's set to `false`!) and wondering why it doesn't work.

Reply:
Well, what would it take for them to learn how coercion works in JavaScript?

Reply:
"Explicit is better than implicit." (even though that's from the zen of python and we have javascript here ;))

Reply:
There's another saying: 'When in Rome, do as the Romans do.' This is JavaScript. Coercion is everywhere. You're not escaping that. Gotta learn at some point.